src/app/SIPUserAgents/SIPB2BUserAgent.cs

```diff
@@ -72,16 +72,21 @@ public SIPRequest Call(SIPCallDescriptor sipCallDescriptor, SIPEndPoint serverEn
             m_uac.CallRinging += (uac, resp) => CallRinging?.Invoke(uac, resp);
             m_uac.CallAnswered += ClientCallAnswered;
 
-            return m_uac.Call(m_uacCallDescriptor);
+            return m_uac.Call(m_uacCallDescriptor);
+        }
+
+        public void AcknowledgeCall(SIPResponse sipResponse, string content, string contentType)

```

Comment:
Would a better name be AckAnswer? Acknowledge conveys its an info response whereas in fact this is the request that completes the invite transaction.